python/lsst/pex/config/config.py

```diff
@@ -237,23 +237,26 @@ def save(self, outfile, instance):
         """
         value = self.__get__(instance)
         fullname = _joinNamePath(instance._name, self.name)
+
+        # write full documentation string as comment lines (i.e. first character is #)
+        doc = "# " + re.sub(r'\n', r'\n# ', str(self.doc))
         if isinstance(value, float) and (math.isinf(value) or math.isnan(value)):
             # non-finite numbers need special care
-            print >> outfile, "%s=float('%r')"%(fullname, value)
+            print >> outfile, "%s\n%s=float('%r')\n"%(doc, fullname, value)
```

Comment:
I agree, and if it was isolated to `obs_subaru` I would just go ahead and make the changes.  However, I have seen print statements in a number other `obs_` packages, so the abuse is a bit more wide-spread than I'm sure you'd like!
E.g.
https://github.com/lsst/obs_lsstSim/blob/master/config/processCcd.py
https://github.com/lsst/obs_sdss/blob/master/config/sourceAssoc.py